src/Helper/EmbedHelper.php

```diff
@@ -310,7 +300,7 @@ public function handleForm(Form $form, callable $handlerCallback, $formTargetRou
 
             $viewVars['messages'] = [];
             if ($request->hasPreviousSession()) {
-                $messages = $this->session->getFlashBag()->get($formId);
+                $messages = $this->getSession()->getFlashBag()->get($formId);
```

Comment:
Ik vind het beter om hier gewoon `$this->requestStack->getSession()` te gebruiken.

De method die jij hebt toegevoegd vangt een exception af en doet er niks mee, dus wordt het heel lastig te achterhalen wat er gebeurt. De method returned dan `null` en je krijgt een error op trying to call `getFlashBag()` on `null`...

Reply:
Ah ja zit wat in. Ik had eerst in de constructor de regel `$this->session = $requestStack->getSession()` maar dat geeft dus direct een exceptie in bijv. de console